app/addons/documents/doc-editor/components.react.jsx

```diff
@@ -218,44 +218,137 @@ var AttachmentsPanelButton = React.createClass({
     };
   },
 
+  getInitialState: function () {
+    return {
+      filter: '',
+      maxHeight: 'none'
+    };
+  },
+
+  componentDidMount: function () {
+    $(window).on('resize.attachment-filter', this.updateMaxHeight);
+    Stores.attachmentFilterStore.on('change', this.onChange);
+  },
+
+  componentWillUnmount: function () {
+    $(window).off('resize.attachment-filter', this.updateMaxHeight);
```

Comment:
the results of `updateMaxHeight` etc are not used later (they are put into a ref) so i am assuming it is a leftover. the function can get removed together with the references to `maxHeight` and state.maxheight